### ISSUE TITLE
Indent error message uses wrong token type

### ISSUE THREAD

**Issue by sukolenvo** [checkstyle/checkstyle#10416]
I have checked docs: https://checkstyle.org/config_misc.html#Indentation
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
/var/tmp $ javac Main.java
# Main.class created

/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name = "Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
         <module name="Indentation">
            <property name="basicOffset" value="4"/>
            <property name="braceAdjustment" value="0"/>
            <property name="caseIndent" value="4"/>
            <property name="throwsIndent" value="8"/>
         </module>
    </module>
</module>

/var/tmp $ cat Main.java
public class Main {
    void test() {
        switch (1) {
        default: test();
        }
    }
}

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.44-all.jar -c config.xml Main.java
Starting audit...
[ERROR] .../Main.java:4:9: 'case' child has incorrect indentation level 8, expected level should be 12. [Indentation]
Audit done.
Checkstyle ends with 1 errors.
```

I am expecting error message to use corrent token type "default" instead of "case", like:
`Main.java:4:9: 'default' child has incorrect indentation level 8, expected level should be 12. [Indentation]`
